Distance_3/benchmark/Distance_3/bench_thingi10k_3.cpp

```diff
@@ -68,7 +68,7 @@ struct Test
         nb_closed_pairs++;
       }
     };
-    
+
```

Comment:
Concerning trailing whitespace I can only recommend that you pass to vscode and install the plugin that removes whitespace on saving a file